x-pack/solutions/observability/plugins/streams/server/routes/streams/crud/route.ts

```diff
@@ -166,11 +166,23 @@ export const editStreamRoute = createServerRoute({
 
     if (
       isGroupStreamDefinition({ ...params.body.stream, name: params.path.name }) &&
-      params.path.name.startsWith('logs')
+      params.path.name.startsWith('logs.')
     ) {
-      throw badRequest('A group stream name can not start with logs.');
+      throw badRequest('A group stream name can not start with [logs.]');
     }
 
+    const body = isGroupStreamDefinitionBase(params.body.stream)
```

Comment:
To make sure I get it right - this branch is just to make typescript happy, right?

Reply:
Yes